README.md

```diff
@@ -82,10 +82,9 @@ contribute in some way.
   Documentation powered by Netlify.
 </p>
 
-### Contributing
+###  For Contribution
 
-
-**[GitHub repository](https://github.com/creativecommons/vue-vocabulary)**
```

Comment:
👍

Reply:
why we need a link of same repo when we are already on it?

Reply:
understood